llvm/docs/RISCVUsage.rst

```diff
@@ -329,6 +329,9 @@ The primary goal of experimental support is to assist in the process of ratifica
 ``experimental-smctr``, ``experimental-ssctr``
   LLVM implements the `1.0-rc3 specification <https://github.com/riscv/riscv-control-transfer-records/releases/tag/v1.0_rc3>`__.
 
+``experimental-xqcicsr``
```

Comment:
This should be in the "Vendor Extensions" section further down

Reply:
Thanks. I've moved it to the Vendor Extensions section.